ui/intermittent-failures/GraphAlternateView.jsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable array-callback-return */
+
+import React from 'react';
+import ReactTable from 'react-table';
+
+const GraphAlternateView = ({ graphData, colNum }) => {
+  const columnsOne = [
+    {
+      Header: 'Date',
+      accessor: 'x',
+    },
+    {
+      Header: 'Failure Count per Push',
+      accessor: 'y',
+    },
+  ];
+
+  const columnsTwo = [
+    {
+      Header: 'Date',
+      accessor: 'x',
+    },
+    {
+      Header: 'Failure Count',
+      accessor: 'y',
+    },
+    {
+      Header: 'Push Count',
+      accessor: 'z',
+    },
+  ];
+
+  const alternateGraph = {
+    column: [],
+    data: [],
+  };
+
+  const convertDateToString = date =>
+    date.toLocaleDateString('en-US', {
```

Comment:
The dates should be in UTC.

Reply:
Do we have a time->utc somewhere already in this project?

Reply:
It's listed in the page title, so yes, it is being used elsewhere.

Reply:
@yogmel The `moment` library is used in Intermittent Failures View to format dates and I think it's simpler to format in UTC time that way.